.github/actions/spelling/excludes.txt

```diff
@@ -37,27 +46,95 @@ ignore$
 \.lock$
 \.map$
 \.min\..
+\.mo$
 \.mod$
 \.mp[34]$
 \.o$
 \.ocf$
 \.otf$
+\.p12$
+\.parquet$
 \.pdf$
 \.pem$
+\.pfx$
 \.png$
 \.psd$
 \.pyc$
 \.pylintrc$
+\.qm$
 \.s$
+\.sig$
+\.so$
 \.svgz?$
+\.sys$
 \.tar$
+\.tgz$
 \.tiff?$
 \.ttf$
 \.wav$
 \.webm$
 \.webp$
 \.woff2?$
+\.xcf$
 \.xlsx?$
+\.xpm$
+\.xz$
 \.zip$
+^localization/japanese/
 ^\.github/actions/spelling/
+^\Q.github/reviewers.yml\E$
 ^\Q.github/workflows/spelling.yml\E$
+^\Qcommon/cultures/cwp_aboriginal_cultures.txt\E$
+^\Qcommon/cultures/cwp_african_horn_cultures.txt\E$
```

Comment:
Most of these files appear to be mostly names by culture, and thus there's nothing really worth checking.